### PR TITLE
new tutorial: preparing meshes using FreeCAD/OpenSCAD and Gmsh

### PR DESCRIPTION
**Pull request by vlukes**

### CODE REVIEW

File: script/convert_mesh.py [hunk 1]
```diff
@@ -29,6 +30,7 @@
     'refine' : 'uniform refinement level [default: %(default)s]',
     'format' : 'output mesh format (overrides filename_out extension)',
     'list' : 'list supported readable/writable output mesh formats',
+    'merge' : 'remove duplicated verticies',
```
Comment (rc) [inline script/convert_mesh.py]: remove duplicate vertices

File: script/convert_mesh.py [hunk 2]
```diff
@@ -113,6 +117,26 @@ def main():
...
+            new_coors = coors[vidxs, :]
+            remap[vmap[:, 1]] = remap[vmap[:, 0]]
+            desc = mesh.descs[0]
+            new_conn = remap[mesh.get_conn(desc)]
+            new_ngroups = mesh.cmesh.vertex_groups[vidxs]
+
+            mesh = Mesh.from_data(mesh.name + '_merged',
+                                  new_coors, new_ngroups,
+                                  [new_conn], [mesh.cmesh.cell_groups], [desc])
+
```
Comment (rc) [inline script/convert_mesh.py]: Would `fix_double_nodes()` work here?

**Comment by rc** [comment]
Thanks, great tutorial! Just address the above comments, and fix `Pyton` in the scheme :) Then some minor tweaks/fixes:

a quite simple Python scripts -> simple Python scripts
of 'MeshPart -> of `MeshPart` ?
FreeCAD can be uses as -> FreeCAD can be used as
the Gmsg graphical interface -> the Gmsh graphical interface
giving `-merge` (or just '-m') -> giving `--merge` (or just `-m`)

Well done!

File: script/convert_mesh.py (hunk 1 above)
Comment (vlukes) [inline script/convert_mesh.py]: fixed

File: script/convert_mesh.py (hunk 2 above)
Comment (vlukes) [inline script/convert_mesh.py]: Yes it works. I use it now, but I would like to reimplement fix_double_nodes() using cKDTree.

**Comment by vlukes** [comment]
Corrected, thank you for your comments.

File: script/convert_mesh.py (hunk 1 above)
Comment (rc) [inline script/convert_mesh.py]: Not yet ;) verticies -> vertices

File: script/convert_mesh.py (hunk 2 above)
Comment (rc) [inline script/convert_mesh.py]: Yes, if it's faster/more memory efficient.

**Comment by rc** [comment]
Almost - the "Pyton" needs to be fixed in images/preprocessing/workflow.png and verticies -> vertices. Thanks!

**Comment by vlukes** [comment]
Now?

**Comment by rc** [comment]
LGTM!

**Pull request merged**